components/molecules/HoverCardWrapper/hover-card-wrapper.tsx

```diff
@@ -5,6 +5,7 @@ import { useSingleContributor } from "lib/hooks/useSingleContributor";
 import ContributorHoverCard, { ContributorsProfileType } from "../ContributorHoverCard/contributor-hover-card";
 
 import roundedImage from "lib/utils/roundedImages";
+import { TooltipProvider } from "@radix-ui/react-tooltip";
```

Comment:
We already have a `TooltipProvider` in the root of the app, if you need this for the Storybook story to work, wrap the component in the story itself.

Reply:
We can revisit this if we need to customize the behavior at a more granular level